runtime/ioext/ioext.go

```diff
@@ -0,0 +1,24 @@
+package ioext
+
+import (
+	"io"
+)
+
+// ReadSeekCloser implements io.Reader, io.Seeker, and io.Closer. It is trivially implemented by os.File.
+type ReadSeekCloser interface {
+	io.Reader
+	io.Seeker
+	io.Closer
+}
+
+func NopCloser(r io.Reader) ReadSeekNopCloser {
```

Comment:
Sure, I was just commenting things I saw :)
bad habit... Should have focused on high-level stuff.. Which seems okay,
well, there is some in error handling.

--
Regards Jonas Finnemann Jensen.

2016-03-25 11:45 GMT-07:00 Brian Stack <notifications@github.com>:

> In runtime/ioext/ioext.go
> <https://github.com/taskcluster/taskcluster-worker/pull/55#discussion_r57476233>
> :
>
> > @@ -0,0 +1,24 @@
> > +package ioext
> > +
> > +import (
> > +	"io"
> > +)
> > +
> > +// ReadSeekCloser implements io.Reader, io.Seeker, and io.Closer. It is trivially implemented by os.File.
> > +type ReadSeekCloser interface {
> > +	io.Reader
> > +	io.Seeker
> > +	io.Closer
> > +}
> > +
> > +func NopCloser(r io.Reader) ReadSeekNopCloser {
>
> Haha, yes. I'll circle back and get these things before we do a real
> review. Should've defined "30% review" before I asked for one. I just use
> it to mean something between a design review and checking to see if
> everything is documented correctly [image: :smile:]. There's still a
> great bit of work to do before this is ready to be landed.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/taskcluster/taskcluster-worker/pull/55/files/641d4358cbd8bfb1d63a428002d6cd29f2372f18#r57476233>
>